### PR TITLE
FTP: Add a setting to enable automatic server encoding detection

### PR DESCRIPTION
**Pull request by TheDeadOne**
References #2998

I've added a method to FtpSettings and FtpsSettings to enable or disable automatic server encoding detection.

### CODE REVIEW

File: ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala [hunk 1]
```diff
@@ -123,6 +130,7 @@ final class FtpSettings private (
     s"credentials=$credentials," +
     s"binary=$binary," +
     s"passiveMode=$passiveMode," +
+    s"autodetectUTF8=$autodetectUTF8" +
```
Comment (ennru) [inline ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala:133]:
```suggestion
    s"autodetectUTF8=$autodetectUTF8," +
```

File: ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala [hunk 2]
```diff
@@ -223,6 +239,7 @@ final class FtpsSettings private (
...
```
Comment (ennru) [inline ftp/src/main/scala/akka/stream/alpakka/ftp/model.scala:242]:
```suggestion
    s"autodetectUTF8=$autodetectUTF8," +
```

**Comment by TheDeadOne** [comment]
@ennru thank you for correcting the result of my inattention.

**Review by ennru — APPROVED** [review]
LGTM.
Thank you!

**Pull request merged**